prboom2/src/p_saveg.c

```diff
@@ -559,7 +559,8 @@ enum {
   tc_scroll,      // killough 3/7/98: new scroll effect thinker
   tc_pusher,      // phares 3/22/98:  new push/pull effect thinker
   tc_flicker,     // killough 10/4/98
-  tc_endspecials
+  tc_endspecials,
+  tc_friction // store friction for cl 9
```

Comment:
We could add this before the `tc_endspecials` value, but that value is always used with an equality check, so it doesn't need to be at the end. If we changed the value of `tc_endspecials` then all old save files would be broken when loaded in this version (the end marker would be interpreted as friction instead and the load process would error out).